docs/architecture.md

```diff
@@ -0,0 +1,64 @@
+# Architecture
+
+This documents the desired architecture for this project. This was initially discussed in [this issue](https://github.com/TwinePlatform/DataPower/issues/212).
+
+### Root
+```
+./
+|- config       Project configuration
+|- db           Database migration files
+|- docs         Documentation
+|- client       Client app
+|- scripts      Developer and deployment scripts
+|- server       Server app
+|- tests        ...Tests
+|- ...dotfiles
+|- package.json Single top-level package.json instead of two different ones
+|- README.md
+```
+
+### Client
+```
+./client
+|- assets             Raw assets (img, external or pre-compiled js/css, etc.)
+|- dist               Built assets will be stored and served from here
+|- css                Styling source files
```

Comment:
is this dependant on whether we use styled elements?

Reply:
Yes, if we use styled elements, this will probably disappear.